package.json

```diff
@@ -101,5 +101,9 @@
     "stylelint-config-standard": "^34.0.0",
     "stylelint-scss": "^5.2.1",
     "vitest": "^1.4.0"
+  },
+  "overrides": {
```

Comment:
Perhaps leave a comment about why overrides were necessary. I remember these were in the deps before but you had to move them here because tests were failing, correct?

Reply:
These are the dependencies of dependencies, which I can't directly manage. fast-xml-parser is a dependency of s3rver, and micromatch are used by another package.

overrides is used for dependencies which I can't upgrade directly